### PR TITLE
refactor(gateway): remove the `$` prefix

### PR DESCRIPTION
**Pull request by MaskDuck**
## Summary

https://github.com/discord/discord-api-docs/pull/5067
## Checklist

- [ ] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
  - [ ] I have run `task pyright` and fixed the relevant issues.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by DenverCoder1 — APPROVED** [review]
Looks good, I haven't tested

**Review by Skelmis — CHANGES_REQUESTED** [review]
The discord change you linked modifies 3 fields, you've changed 5. Care to explain?

**Comment by MaskDuck** [comment]
@Skelmis 
image

**Comment by MaskDuck** [comment]
or should i remove two of them too?

**Comment by Skelmis** [comment]
And..?

Can you provide some links as to where it said those fields also required changing, I can't find them.

**Comment by MaskDuck** [comment]
i think we have to remove them since...
https://github.com/discord/discord-api-docs/search?q=referrer&type=code
https://github.com/discord/discord-api-docs/search?q=referring_domain&type=code

both of those returns None.

File: nextcord/gateway.py [hunk 1]
```diff
@@ -420,13 +420,7 @@ async def identify(self):
...
             "d": {
                 "token": self.token,
-                "properties": {
-                    "$os": sys.platform,
-                    "$browser": "nextcord",
-                    "$device": "nextcord",
-                    "$referrer": "",
-                    "$referring_domain": "",
-                },
+                "properties": {"os": sys.platform, "browser": "nextcord", "device": "nextcord"},
```
Comment (DenverCoder1) [inline nextcord/gateway.py:423]: I'd recommend to keep the comma after the device name so it displays on separate lines

```suggestion
                "properties": {
                    "os": sys.platform,
                    "browser": "nextcord",
                    "device": "nextcord",
                },
```

**Review by DenverCoder1 — APPROVED** [review]
:+1: Should be fine to remove referrer stuff, as mentioned it doesn't appear in the docs. Only references I could find were in discord.py and DiscordPHP. I checked Discord.js and it seems to not have them.

**Review by Skelmis — APPROVED** [review]

**Review by ooliver1 — APPROVED** [review]

**Pull request merged**